pkg/config/config_test.go

```diff
@@ -367,6 +369,57 @@ func (s *ConfigSuite) TestCalculateCollectorURL(c *C) {
 	}
 }
 
+func (s *ConfigSuite) TestCalculateDimensionalMetricURL(c *C) {
+	testCases := []struct {
+		name         string
+		license      string
+		collectorURL string
+		staging      bool
+		want         string
+	}{
+		{
+			"Default URL, no region license, no collector URL",
+			"0123456789012345678901234567890123456789",
+			"",
+			false,
+			"https://metric-api.newrelic.com",
+		},
+		{
+			"Staging URL, no region license, no collector URL",
+			"0123456789012345678901234567890123456789",
+			"",
+			true,
+			"https://staging-metric-api.newrelic.com",
+		},
+		{
+			"Default URL, eu license region, no collector URL",
+			"eu01xx6789012345678901234567890123456789",
+			"",
+			false,
+			"https://metric-api.eu.newrelic.com",
+		},
+		{
+			"Staging URL, eu license region, no collector URL",
+			"eu01xx6789012345678901234567890123456789",
+			"",
+			true,
+			"https://staging-metric-api.eu.newrelic.com",
+		},
+		{
+			"From Collector URL",
+			"gov01x6789012345678901234567890123456789",
+			"https://metric-api.test",
```

Comment:
Should we have one more test here where `collectorURL` is blank but `license` is `gov01x...` so `want` should be blank? Or instead of blank would it be the default value of the `collectorURL` for gov?

Reply:
In that case I think it should use the default value for the metric api: https://metric-api.newrelic.com

Can you confirm if is the correct behaviour? If so, I will add a test case

Reply:
In the case here if a `gov` license is provided then the API domain that should be used is the FedRAMP one. If we default to `metric-api.newrelic.com` customers who use a `gov` license will get the incorrect domain.

Reply:
Done.
Good catch!